app/lib/widgets/action/action_result_screen.dart

```diff
@@ -39,7 +41,13 @@ class _ActionResultScreenState extends State<ActionResultScreen> {
         ? Future.value(
             ActionResult.fromPatient(widget.patient, widget.performedActionId))
         : ActionService.fetchActionResult(
-            widget.patient, widget.performedActionId);
+                widget.patient, widget.performedActionId)
+            .then((value) {
+            if (widget.onActionResultAvailable != null) {
+              widget.onActionResultAvailable!(value?.patient);
```

Comment:
Here the syntactic sugar might be nice, but fine as is.
```dart
widget.onActionResultAvailable?.call(value?.patient);
```

Reply:
Done